### ISSUE TITLE
formatter: `sortImports` treats internal typescript paths/aliases as `external`

### ISSUE THREAD

**Issue by james-camilleri** [oxc-project/oxc#25795]
### Summary

`oxfmt` treats typescript `path` aliases as external packages when sorting, whereas they are actually internal. The `tsconfig-path` option in `perfectionist/sort-imports` does not exist in oxfmt, and the `subpath` option only works with native node import maps.

#### Input
```tsx
import { external } from 'external-package'
import { internal_1 } from 'alias/internal'
import { internal_2 } from './subpath/internal'
```

#### Oxfmt Config
```jsonc
{
    "sortImports": {
        "newlinesBetween": true,
        "groups": [
            "external",
            "internal",
            // This doesn't work either.
            // ["internal", "subpath"]
        ]
    },
}
```

#### Expected Output
```ts
import { external } from 'external-package'

import { internal_1 } from 'alias/internal'
import { internal_2 } from './subpath/internal'
```

#### Actual Output
```ts
import { external } from 'external-package'
import { internal_1 } from 'alias/internal'

import { internal_2 } from './subpath/internal'
```
*(Oxfmt version: `0.61.0`)*

Presumably `eslint-import/sort-imports` can do this since it has type information, which `oxfmt` doesn't have, but `eslint-perfectionist/sort-imports` manages it with a reference to the tsconfig file and no additional type info.

I understand that this is probably not a high priority, especially since a native replacement for typescript path aliases is now widely available, but it is a hitch in a seamless transition from prettier+eslint to oxfmt+oxlint, so perhaps it ought to be mentioned in the docs explicitly that it isn't supported if it's not something that can be fixed.

**Comment by leaysgur** [comment]
Thank you for the report.

And yes, I would like to handle this on the documentation side.

Also as a workaround, I think `internalPattern` or `customGroups` can be used.

```json
{
  "sortImports": {
    "newlinesBetween": true,
    "internalPattern": ["alias/"],
    "groups": [
      "external",
      "internal",
      { "newlinesBetween": false },
      ["parent", "sibling", "index"]
    ]
  }
}

// or

{
  "sortImports": {
    "newlinesBetween": true,
    "groups": [
      "external",
      "my-alias",
      { "newlinesBetween": false },
      ["parent", "sibling", "index"]
    ],
    "customGroups": [
      { "groupName": "my-alias", "elementNamePattern": ["alias/**"] }
    ]
  }
}
```
